chroot_setup.sh

```diff
@@ -389,7 +389,7 @@ shift
 case "$#" in
    0) # Only the chroot directory was provided: Ask if minimal with includes chroot is desired.
       if (( $isYes == 0 )) ; then
-        echo "Would you like to create a minimal chroot w/ includes @ $CHROOT_DIR? [y/N]: \c"
+        echo "Would you like to create a minimal chroot w/ includes and nls @ $CHROOT_DIR? [y/N]: \c"
```

Comment:
```suggestion
        echo "Would you like to create a default chroot environment @ $CHROOT_DIR? [y/N]: \c"
```

I think the includes/nls stuff is probably too much information here for most people. For the newbies, giving this level of detail just adds noise since they likely don't know enough to even know what's best for them - thus good defaults! For experienced users, they likely already know what's in there (or know where to find out).

Reply:
I agree with the simplified prompt